resources/CMakeLists.txt

```diff
@@ -111,6 +111,11 @@ set(ENERGYPLUS_OUTPUTS "")
 set(model_resources_src
   model/OpenStudio.idd
   model/schedulefile.csv
+
+  model/two_stories_pre_intersect.osm
+  model/floorplan_school.osm
+  model/7-7_Windows_Complete.osm
```

Comment:
`7-7_Windows_Complete.osm`  appears unused anywhere in the code.

Reply:
There's something that needs cleanup here: https://github.com/NREL/OpenStudio/blob/c0e8704114bbc582d1e9180b89bb853045ca6930/src/model/test/Space_GTest.cpp#L2576-L2578